src/components/modal/v2/parts/Header.jsx

```diff
@@ -68,7 +78,7 @@ const Header = ({ headline, subheadline, className = '', logo, contentWrapper, c
                 </div>
                 <div className="header__content">
                     <h1>{headline}</h1>
-                    <h2>{subheadline}</h2>
+                    {isQualifying === true ? <h2>{qualifyingSubheadline ?? subheadline}</h2> : <h2>{subheadline}</h2>}
```

Comment:
```suggestion
                    {isQualifying === 'true' && qualifyingSubheadline !== ''  ? <h2>{qualifyingSubheadline}</h2> : <h2>{subheadline}</h2>}
```
So it turns out it was fine to have `isQualifying` be a string instead of a boolean. It's just `qualifyingSubheadline ?? subheadline` part of the check was initially failing and changing it to a boolean "fixed" this aspect of it. Instead the above suggestion addresses both concerns.

Reply:
updated